src/components/elements/Inputs/Map/RHFMap.tsx

```diff
@@ -46,6 +69,57 @@ const RHFMap = ({
       cacheTime: 0
     }
   );
+
+  const refetchData = () => {
+    reloadProjectPolygonData();
+    mapFunctions?.onCancel(polygonDataMap);
+  };
+
+  const {
+    data: projectPolygon,
+    refetch: reloadProjectPolygonData,
+    isRefetching
+  } = useGetV2TerrafundProjectPolygon(
+    {
+      queryParams: {
+        entityType: entity?.entityName || "",
+        uuid: entity?.entityUUID || ""
+      }
+    },
+    {
+      enabled: !!entity,
+      staleTime: 0,
+      cacheTime: 0
+    }
+  );
+  const setBbboxAndZoom = async () => {
+    if (projectPolygon?.project_polygon?.poly_uuid) {
+      const bbox = await fetchGetV2TerrafundPolygonBboxUuid({
+        pathParams: { uuid: projectPolygon.project_polygon?.poly_uuid || "" }
```

Comment:
`??`, although I'm wondering: do you really want to call this API if there is no poly_uuid available? If not, this might be better accomplished using `enabled: projectPolygon.project_polygon?.poly_uuid != null` on the query arguments, or something similar as in the case above.

Reply:
The if above verifies if its null. but the default validation (??) is added anyway.